internal/setup/setup.go

```diff
@@ -664,15 +714,21 @@ func Select(release *Release, slices []SliceKey) (*Selection, error) {
 		for newPath, newInfo := range new.Contents {
 			if old, ok := paths[newPath]; ok {
 				oldInfo := old.Contents[newPath]
-				if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath) && new.Package != old.Package {
+				if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath || newInfo.Kind == GeneratePath) && new.Package != old.Package {
 					if old.Package > new.Package || old.Package == new.Package && old.Name > new.Name {
 						old, new = new, old
 					}
 					return nil, fmt.Errorf("slices %s and %s conflict on %s", old, new, newPath)
 				}
-				continue
+			} else {
+				paths[newPath] = new
+			}
+			switch newInfo.Generate {
+			case GenerateNone, GenerateManifest:
```

Comment:
No, we don't. Because `GenerateNone` would be the cases where `generate` is not specified/used. Using `generate: ""` actually does nothing and will not be a `GeneratePath` either.

As for the tests, every test with `generate: "x"` is actually this case.

Reply:
Right, so because of zero values in Go, there is no way of knowing if a path is `generate: ""` or nothing at all?

Reply:
You are right. Unless we are using pointers. For example, for `TextPath`, you can have `text: ""`.